### PR TITLE
Companion for: `try-runtime::fast-forward`

### PR DESCRIPTION
**Pull request by pmikolajczyk41**
companion for https://github.com/paritytech/substrate/pull/12896

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/2276869

File: polkadot-parachain/src/command.rs [hunk 1]
```diff
@@ -38,6 +38,9 @@ use sp_core::hexdisplay::HexDisplay;
 use sp_runtime::traits::{AccountIdConversion, Block as BlockT};
 use std::{net::SocketAddr, path::PathBuf};
 
+#[cfg(feature = "try-runtime")]
+use try_runtime_cli::block_building_info::timestamp_with_aura_info;
```
Comment (kianenigma) [inline polkadot-parachain/src/command.rs:42]: same.

File: parachain-template/node/src/command.rs [hunk 2]
```diff
@@ -14,6 +14,9 @@ use sc_service::config::{BasePath, PrometheusConfig};
 use sp_core::hexdisplay::HexDisplay;
 use sp_runtime::traits::{AccountIdConversion, Block as BlockT};
 
+#[cfg(feature = "try-runtime")]
+use try_runtime_cli::block_building_info::timestamp_with_aura_info;
```
Comment (kianenigma) [inline parachain-template/node/src/command.rs:18]: same.

**Review by kianenigma — APPROVED** [review]

File: polkadot-parachain/src/command.rs (hunk 1 above)
Comment (pmikolajczyk41) [reply polkadot-parachain/src/command.rs]: moved

File: parachain-template/node/src/command.rs (hunk 2 above)
Comment (pmikolajczyk41) [reply parachain-template/node/src/command.rs]: moved

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/2343409

**Review by sam0x17 — APPROVED** [review]

**Review by Ank4n — APPROVED** [review]

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/2381015

**Comment by ggwpez** [comment]
Merged into https://github.com/paritytech/cumulus/pull/2213

**Pull request closed without merging**